server.go

```diff
@@ -85,28 +109,28 @@ func NewServer(host, port string) *Server {
 	return s
 }
 
-func (this *Server) Serve(method, urlPath string, middlewares ...Middleware) {
+func (s *Server) Serve(method, urlPath string, middlewares ...Middleware) {
```

Comment:
lets do this `this` replacements in a separate PR which we can quickly merge. that would make the diff much more readable.